examples/working_with_features.py

```diff
@@ -0,0 +1,76 @@
+# Copyright (c) 2022 Contributors to the Eclipse Foundation
+#
+# See the NOTICE file(s) distributed with this work for additional
+# information regarding copyright ownership.
+#
+# This program and the accompanying materials are made available under the
+# terms of the Eclipse Public License 2.0 which is available at
+# http://www.eclipse.org/legal/epl-2.0
+#
+# SPDX-License-Identifier: EPL-2.0
+
+from base64 import encode
+import time, sys
+
+from ditto.client import Client
+from ditto.model.feature import Feature
+from ditto.model.namespaced_id import NamespacedID
+from ditto.model.thing import Thing
+from ditto.protocol.envelope import Envelope
+from ditto.protocol.things.commands import Command
+
+
+client = Client()
+client.connect("localhost")
+
+def create_or_modify_feature(client):
+       
+    # Define the feature to be created or how you want it to be after modification
+    # You can provide a semantic definition of your feature
+    myFeature = Feature().with_definition_from("my.model.namespace:FeatureModel:1.0.0").with_property("myProperty", "myValue")
```

Comment:
Think about having the commonly used strings as private constants defined for the example - test namespaces/definition IDs, entity name, feature ID, etc.